tests/lustre/typechecking/shouldFail/const_02.lus

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Right now the logic is to overwrite the previous definition.
I believe we want to make duplicate declarations for constants fail then?

Reply:
If the logic is to overwrite, why it is not returning a type checking error? The second declaration has type `bool`. I guess the behavior is to ignore any subsequent declaration. After this comment, I double-checked and I saw this is the behavior of the old type checker. I find this behavior misleading. Lustre v6 compiler complains about `identifier already declared`. I prefer this more restricted behavior because it is consistent with the handling of duplicated type/node declarations (Kind 2 complains about that ones), it avoids misunderstandings, and I don’t think we need more flexibility for declaring constants/types.

Reply:
I see. Then it is my implementation bug :D 
Yup. Sounds good. I can make it a strict behaviour then and fail if we have duplicate declarations.

Reply:
This is fixed now.